drivers/usb/serial/option.c

```diff
@@ -2183,6 +2183,13 @@ static int option_probe(struct usb_serial *serial,
 			return -ENODEV;
 		}
 	}
+
+	// USB autosuspend
+	if (serial->dev->descriptor.idVendor == cpu_to_le16(0x2C7C)) {
+		pm_runtime_set_autosuspend_delay(&serial->dev->dev, 3000);
```

Comment:
This delay probably can be changed to something smaller, greater than 0, as 0 appears to disable the autosuspend (contrary to what the Linux docs say)

Reply:
We should create a task to explore this value.

Reply:
For now I mentioned it in this comment https://tiermobility.atlassian.net/browse/HI-7?focusedCommentId=90426

IMO the parent task is too big and it needs to be split anyway.